lib/modules/datasource/repology/index.ts

```diff
@@ -177,12 +177,7 @@ export class RepologyDatasource extends Datasource {
       throw err;
     }
 
-    logger.debug(
-      { repoName, pkgName },
-      'Repository or package not found on Repology',
-    );
-
-    return undefined;
+    throw new Error(`Repository or package not found on Repology: ${repoName}/${pkgName}`);
```

Comment:
```suggestion
    throw err;
```

Reply:
But this does not work as there is no `err`.

`TS2304: Cannot find name err`

Reply:
How is this code reached and for what reasons?

Reply:
I looked and there's very few manually thrown errors in datasources. Ideally we shouldn't invent a new type of error here

Reply:
Before there was a ``````logger.debug` which I changed to `logger.warn` to make the problem visible in the logs. See above. May be switch back to the first solution?